database/src/main.cpp

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+#include  "sqlite_database.hpp"
+ 
+ int main(int argc, char **argv){
+    Sqlite_db db;
+    db.create();
```

Comment:
adding a CRUD demo would be more helpful here!
But still...... Great demonstration of SQLITE setup with C++

Reply:
@baimamboukar 

CRUD is planned and in progress...
The first challenge was to provide an API description using Abstract classes in C++.
If you look at the definition of the abstract class you will find that CRUD is planned.
:relaxed: